starport/services/network/networkchain/init.go

```diff
@@ -111,3 +116,22 @@ func (c *Chain) checkGenesis(ctx context.Context) error {
 	// example: gentxs formats are not checked
 	// to perform a full validity check of the genesis we must try to start the chain with sample accounts
 }
+
+// checkGentxFiles checks if a gentx exist for the chain
+func (c *Chain) checkGentxFiles() error {
+	gentxPath, err := c.DefaultGentxPath()
+	if err != nil {
+		return err
+	}
+	if _, err = os.Stat(gentxPath); !os.IsNotExist(err) {
+		return fmt.Errorf("gentx %s found in initial genesis", gentxPath)
+	}
+	gentxsPath, err := c.GentxsPath()
+	if err != nil {
+		return err
+	}
+	if xfilepath.IsEmptyDir(gentxsPath) {
+		return fmt.Errorf("gentxs folder %s is not empty in the initial genesis", gentxsPath)
```

Comment:
The error says the folder is not empty but error will be returned in case if folder is empty according to the check.